src/config/2021.json

```diff
@@ -279,6 +279,14 @@
       "avatar_url": "https://avatars.githubusercontent.com/u/35119235?v=4&s=200",
       "github": "ashleyish"
     },
+    "azakai": {
+      "name": "azakai",
+      "teams": [
+        "reviewers"
+      ],
+      "avatar_url": "https://avatars.githubusercontent.com/u/451275?v=4&s=200",
+      "github": "azakai"
+    },
```

Comment:
@azakai FYI adding your contributor info. Open a new PR to make any changes.

cc @RReverser not sure what full name to put here